src/gui/Clipboard.cpp

```diff
@@ -38,10 +41,18 @@ void Clipboard::setText(const QString& text)
 {
     QClipboard* clipboard = QApplication::clipboard();
 
+#ifdef Q_OS_MAC
+    new MacPasteboard;
```

Comment:
What does this do? Is this object ever deleted?

Reply:
It inits the override methods. I added it to the constructor and now it is properly deleted at the destructor.

Reply:
I would prefer you make it a QScopedPointer.

Reply:
@phoerious fixed